python/py-tmuxp/Portfile

```diff
@@ -61,22 +52,38 @@ if {${name} ne ${subport}} {
              ${worksrcpath}/${subport}
     }
 
-    post-build {
-        system -W ${worksrcpath}/docs "make PYTHONPATH=${worksrcpath}/build/lib SPHINXBUILD=${prefix}/bin/sphinx-build-${python.branch} text man"
+    pre-destroot {
+        xinstall -d ${destroot}${prefix}/share/examples/${subport}
+        xinstall -m 644 {*}[glob ${worksrcpath}/examples/*] \
+            ${destroot}${prefix}/share/examples/${subport}
     }
 
-    post-destroot {
-        set doc_build_dir ${worksrcpath}/docs/_build
+    variant html_man_docs \
```

Comment:
can we call this variant just `doc` or `docs` as is done in pretty much every other port?

Reply:
Sure, done